packages/xod-client-browser/src/containers/App.jsx

```diff
@@ -46,7 +48,35 @@ class App extends client.App {
       [client.COMMAND.RENAME_PROJECT]: this.props.actions.requestRenameProject,
     };
 
-    props.actions.openProject(props.initialProject);
+    this.urlActions = {
+      [client.URL_ACTION_TYPES.OPEN_TUTORIAL]: this.onOpenTutorial,
+    };
+
+    document.addEventListener('click', (e) => {
```

Comment:
This makes the constructor really long. May I suggest extracting the handler to its own method? So here will left something like `document.addEventListener('click', this.onAnchorClick)`?